src/features/game/toast/ToastManager.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { Panel } from "components/ui/Panel";
+import { ToastContext } from './ToastQueueProvider';
+
+export const ToastManager = () => {
+  const { toastList, removeToast } = useContext(ToastContext);
+  const [listed, setListed] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (toastList.length >= 1) {
+      setListed(true);
+    } else {
+      setListed(false);
+    }
+  }, [toastList]);
+
+  return (
+    <div>
+      {listed &&
+        <div className="bg-brown-600 p-0.5 text-white shadow-lg flex flex-col items-end mr-2 sm:block fixed top-20 left-2 z-[99999]">
+          <Panel>
+            {toastList.map(({ content, id }) => (
```

Comment:
<img width="284" alt="Screen Shot 2022-01-30 at 7 31 16 pm" src="https://user-images.githubusercontent.com/11745561/151720633-ea5525e4-7f99-4087-9f22-c4b9ea10a4cf.png">

The text alignment gets a bit funny when we have 2 different resources subtracting. 

I think we could use some iconography to address this in the future :)